### DISCUSSION TITLE
Access dTextAlphaBits and dGraphicsAlphaBits properties of Ghostscript

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#852] [Help]
I do not find a way to set  `dTextAlphaBits `and `dGraphicsAlphaBits` properties while converting PDF to Image.

The `PdfReadDefines `class has five properties but `dTextAlphaBits `and `dGraphicsAlphaBits` are missing among them. It also has `IEnumerable<IDefine> Defines` property which is read-only and I can't add my own defines. The `PdfReadDefines `
 inherit from DefinesCreator where I find some create defines method but unfortunately these properties are protected.

**Comment by dlemstra** [comment]
The `dTextAlphaBits` and `dGraphicsAlphaBits ` option is set with the `AntiAlias` option of the `ReadSettings`. The default is `true` and then it passes the value `4` when it is false `1` will be passed instead.

Do you need more control over these options?
